--- old_parser.y	1999-06-09 23:26:12.000000000 +0200
+++ ORBit-0.4.93/libIDL/parser.y	2021-01-15 18:26:00.070714851 +0100
@@ -594,7 +594,7 @@
 			{ $<str>$ = "struct"; }
 			z_new_scope_catch '{'		{
 	g_hash_table_insert (__IDL_structunion_ht, $4, $4);
-	$$ = IDL_type_struct_new ($4, NULL);
+	$<tree>$ = IDL_type_struct_new ($4, NULL);
 }				member_list
 			'}' pop_scope			{
 	g_hash_table_remove (__IDL_structunion_ht, $4);
@@ -611,7 +611,7 @@
 				switch_type_spec
 			')' '{'				{
 	g_hash_table_insert (__IDL_structunion_ht, $4, $4);
-	$$ = IDL_type_union_new ($4, $7, NULL);
+	$<tree>$ = IDL_type_union_new ($4, $7, NULL);
 }				switch_body
 			'}' pop_scope			{
 	g_hash_table_remove (__IDL_structunion_ht, $4);
